contrib/html_demo/JupyterCode/4_finetune_on_textile_pattern_dataset.ipynb

```diff
@@ -0,0 +1,759 @@
+{
```

Comment:
<p>[minor] Should we use the same line here as in some other notebooks, so that the user can just run it:</p><pre class="ql-syntax" spellcheck="false">DATA_PATH = unzip_url(Urls.fridge_objects_path, exist_ok=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done.